.github/workflows/test.yml

```diff
@@ -25,7 +26,6 @@ jobs:
       - name: Prepare working environment for running and collecting test results
         run: |
           sudo apt-get update
```

Comment:
if you install nothing why we need this?

Reply:
Good catch. Thanks.

Reply:
Good catch. Thanks!